pkg/utils/mageutils/release.go

```diff
@@ -42,11 +43,11 @@ func CreateLocalTag(tag string) (err error) {
 	return sh.RunV("git", "tag", "-s", tag, "-m", "release "+tag)
 }
 
-// CheckoutRcBranch checkout/create and rc/tag branch based on main branch
-func CheckoutRcBranch(tag string) error {
+// CheckoutReleaseBranch creates if not exists a release branch and then checkout
+func CheckoutReleaseBranch(tag string) error {
 	mg.Deps(isGitExistent)
 
-	branchName := "release/" + tag[:4]
+	branchName := fmt.Sprintf("release/%s", tag[:4])
```

Comment:
This index access can be panic? What do you think about add a validation and panic with a better message?
```go
if len(tag) < 4 {
     panic(fmt.Sprintf("invalid tag format: %s", tag)) // This panic message can be better
}
```

Reply:
changed this check to `isValidTag()` as we may need to use it in another target

Reply:
UpVersion command already contains the branch name.

 `nextReleaseBranchName: represents the next release branch name (release/v1.1).`
 
 ```
// CheckoutReleaseBranch creates if not exists a release branch and then checkout
func CheckoutReleaseBranch(branchName string) error {
	mg.Deps(isGitExistent)

	if err := sh.RunV("git", "checkout", branchName); err != nil {
		log.Printf("First %s release, creating release branch", branchName)

		return sh.RunV("git", "checkout", "-b", branchName)
	}

	return nil
}
```

Reply:
done